bucket/synology-surveillance-station.json

```
@@ -0,0 +1,34 @@
+{
+    "version": "1.2.11-2181",

```

```suggestion
    "version": "1.2.11-2181",
    "description": "Synology Surveillance Station client",
    "homepage": "https://www.synology.com/en-us/surveillance",
```
`description` and `homepage` is required for official buckets.

---

It seems I forgot a push. This one is fixed now.